### PR TITLE
Improve on physics examples

### PR DESCRIPTION
**Pull request by jmiskovic**

### CODE REVIEW

File: examples/Physics/Hand_Physics/main.lua [hunk 1]
```diff
@@ -8,21 +8,21 @@ from hand controller. This results in lousy and unconvincing collisions with oth
 physics engine doesn't know the speed of hand colliders at the moment of collision.
 
 An improvement is to set linear and angular speed of kinematic hand colliders so that they
-approach the target (actual location/orientation of hand controller). This works excellent for one
-controller. When you try to squeeze an object between two hands, physics break. This is because
-kinematic hand controllers are never affected by physics engine and unrealistic material
-penetration cannot be resolved.
+approach the target (actual location/orientation of hand controller). This works well for one
+hand physics, however physics start to glitch when you try to squeeze object between two hands.
```
Comment (bjornbytes) [inline examples/Physics/Hand_Physics/main.lua:12]: Minor suggestion:

> This works well for one hand ~physics~, however physics **will** start to glitch when you try to squeeze **an** object between two hands.

File: examples/Physics/Hand_Physics/main.lua [hunk 2]
```diff
@@ -8,21 +8,21 @@ from hand controller. This results in lousy and unconvincing collisions with oth
...
 
 The approach taken here is to have hand controllers behave as normal dynamic colliders that can be
 affected by other collisions. To track hand controllers, we apply force and torque on collider
 objects that's proportional to distance from correct position.
 
 This means hand colliders won't have 1:1 mapping with actual hand controllers, they will actually
-'bend' under large force. Also the colliders can actually become stuck behind another object. This
-is sometimes frustrating to use, so in this example hand colliders can ghost through objects or
-become solid using trigger button.
+'bend' under large force. Also the colliders can become stuck and burried beneath other objects.
```
Comment (bjornbytes) [inline examples/Physics/Hand_Physics/main.lua:21]: s/burried/buried

File: examples/Physics/Drawing_Colliders/main.lua [hunk 3]
```diff
@@ -1,124 +0,0 @@
---[[ A utility function that draws all possible physics colliders and joint geometry
```
Comment (bjornbytes) [inline examples/Physics/Drawing_Colliders/main.lua:1]: Deprecated in favor of phywire, right?

Comment (bjornbytes) [reply examples/Physics/Drawing_Colliders/main.lua:1]: Just saw commit message

**Comment by jmiskovic** [comment]
Oh, I didn't see your comments until now, sorry...

**Pull request merged**